digitalocean/Tag.py

```diff
@@ -79,29 +80,33 @@ def __remove_resources(self, resources):
         return self.__get_resources(resources, method='DELETE')
 
 
-    def __extract_resources_from_droplets(self, data):
+    def __build_resources(self, data, object_class: str, resource_type: str):
```

Comment:
Was it really necessary to rename the method?

Reply:
> Was it really necessary to rename the method?

Since this method is now reused to build other `resources` fields, yeah it makes sense to rename this method to something more generic, that don't include "droplet" in its name.

Reply:
Got it! :) We can make it even more clear in the docs then!